tikv-configuration-file.md

```diff
@@ -349,9 +349,10 @@ Configuration items related to storage.
 
 ### `scheduler-worker-pool-size`
```

Comment:
This part was updated in [#8644](https://github.com/pingcap/docs-cn/pull/8644) and revised in [#8784](https://github.com/pingcap/docs-cn/pull/8784).